ap.md

```diff
@@ -0,0 +1,236 @@
+{::options parse_block_html="true" /}
+
+<div class="row">
+# Build Your Own Internet
+<div class="large-8 columns">
+
+Along with connecting to an existing Wifi network, Tessel 2 can also act as a portable router and emit a custom Wifi network called an access point. We can use this functionality to connect directly with our Tessel through a web browser. The advantage of doing this through Tessel's access point instead of through a local Wifi network is reduced latency of requests to the Tessel since they won't be making a detour through a different router. This also means there is no need for a local network for connecting to Tessel through the web. By the end of this article, we'll be able to control the Tessel LEDs through a web app served by the Tessel.  
+
+In your command line, make a folder for your Tessel code, then initialize a Tessel project in that folder by running each of the following commands in the terminal:
+
+`mkdir tessel-router`
+
+`cd tessel-router`
+
+`t2 init`
+
+Great! Now you're set up to run code on Tessel. Your "tessel-router" folder now contains a "package.json" with some metadata Node uses for your project, and a file called "index.js".
+</div>
+<div class="large-4 columns right">
+![router](./images/router.jpg)
+</div>
+</div>
+
+<div class="row">
+<div class="large-12 columns">
+
+<hr>
+
+Rename the “index.js” file you’ve just created to “ap.js”, then copy and paste the below script over the existing text:
+
+{% highlight javascript %}
+// Import the interface to Tessel hardware
+var tessel = require('tessel');
+// Load the http module to create an http server.
+var http = require('http');
+
+// Configure our HTTP server to respond with "Hello from Tessel!" to all requests.
+var server = http.createServer(function (request, response) {
+  response.writeHead(200, {"Content-Type": "text/plain"});
+  response.end("Hello from Tessel!\n");
+});
+
+// Listen on port 8080, IP defaults to 192.168.1.101. Also accessible through [tessel-name].local
+server.listen(8080);
+
+// Put a friendly message on the terminal
+console.log("Server running at http://192.168.1.101:8080/");
+
+{% endhighlight %}
+
+Now that our server is in place, let's get our access point set up. In the terminal, run the following command:
+
+`t2 ap -n TesselRouter`
+
+This will make an open wifi network called TesselRouter. If you open the wifi setting of your computer or a separate device like a smartphone or tablet, and scan for new devices, you should be able to see and connect to this new network. After connecting to TesselRouter, run the following command in your terminal:
+
+`t2 run ap.js`
```

Comment:
Actually,  what if this got split out into two pages: one is an introduction to what an AP is, showing how to use the CLI tool (complete with turning it on and off and looking for it in a wifi scanner).

Then the second page would be more about making a webserver on Tessel - the examples you've already written - and it would use the AP command.

Partly, I want to make this distinction because you could use T1 as a webserver too. So looking at your example, I'm not as impressed as I should be because I've already done this stuff without an access point.